### DISCUSSION TITLE
Not allowed to change or delete some files requested

### DISCUSSION THREAD

**Discussion by haqaliz** [deta/cloud-docs#717] [Help]
hey guys,
I'm getting `Error: failed to deploy: Not allowed to change or delete some files requested` while deploying to micro

can some one explain it to men how can I fix it?

**Comment by sudo-subg** [comment]
Can you share the directory structure?

**Reply by haqaliz** [reply]
yeah sure

![Group 1](https://user-images.githubusercontent.com/78667368/211303001-11bf0276-08db-49e4-bf6a-b21d3b2dd1de.png)

**Reply by sudo-subg** [reply]
You need an entrypoint file (in case of Node.js an index.js file) as described [here](https://docs.deta.sh/docs/micros/deploy#nodejs). And Deta Cloud has `nodejs12.x, nodejs14.x` runtimes support only.

**Reply by haqaliz** [reply]
thx I put the enrty point file (index.js) in the src folder
